### ISSUE TITLE
takagg import bug

### ISSUE THREAD

**Issue by SuperJogosDBZ** [MadeBaruna/paimon-moe#71]
the import of excel from the takagg program is not working for me, is it because my excel is in Portuguese?

**Comment by MadeBaruna** [comment]
yes, as stated on the import page, currently only supporting english reward name

**Comment by jungco7** [comment]
I changed all Korean reward names to English, but still does not proceed in "Processing..."
[this file](https://docs.google.com/spreadsheets/d/1i9Kh3hCRXQS6faOX1s3vSVGgp4xgeMsWAiB-EbzCILA/edit?usp=sharing)
